src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/Profile.java

```diff
@@ -50,16 +40,11 @@ private Profile() {
       blavf.setWeight(c, 0d);
     }
     this.linearAvf = blavf.build();
-    this.floorAreaWeightRange = null;
-    this.nbBedroomsWeightRange = null;
-    this.nbSleepingWeightRange = null;
-    this.nbBathroomsWeightRange = null;
-    this.terraceWeightRange = null;
-    this.floorAreaTerraceWeightRange = null;
-    this.wifiSubjectiveValueWeightRange = null;
-    this.pricePerNightSubjectiveValueWeightRange = null;
-    this.nbMinNightSubjectiveValueWeightRange = null;
-    this.teleSubjectiveValueWeightRange = null;
+
+    this.rangeMap = new HashMap<Criterion, Range<Double>>();
```

Comment:
Attention, il y a un warning ici (retirer les arguments redondants).

Reply:
Je ne comprends pas ton commentaire, il ne me semble pas avoir de warning de mon coté

Reply:
Fixed in commit 2d6652c